iModelJsNodeAddon/IModelJsNative.cpp

```diff
@@ -2612,6 +2612,89 @@ struct NativeRevisionUtility : BeObjectWrap<NativeRevisionUtility>
         }
     };
 
+//=======================================================================================
+//  Projects the SchemaUtility class into JS.
+//! @bsiclass
+//=======================================================================================
+struct NativeSchemaUtility : BeObjectWrap<NativeSchemaUtility>
+    {
+    private:
+        DEFINE_CONSTRUCTOR
+
+    public:
+        NativeSchemaUtility(NapiInfoCR info) : BeObjectWrap<NativeSchemaUtility>(info) {}
+        ~NativeSchemaUtility() {SetInDestructor();}
+
+    static Napi::Value ConvertCustomAttributes(NapiInfoCR info)

```

Comment:
Both methods here and in JsInterop are 95% the same.  You should be able to refactor them to share code.  For example make a one method that takes a flag to control if CAs are converted in both classes then have the two exposed Napi methods call one common method with different flags.